modules/thunder_gqls/tests/src/Traits/ThunderGqlsTestTrait.php

```diff
@@ -127,4 +138,17 @@ protected function getExpectedResponseFromFile(string $name): string {
     return file_get_contents($this->getQueriesDirectory() . '/' . $name . '.response.json');
   }
 
+  /**
+   * Decodes a json string.
+   *
+   * @param string $json
+   *   The json string.
+   *
+   * @return mixed
+   *   The decoded json.
+   */
+  protected function jsonDecode(string $json) {
```

Comment:
Could we use Drupal\Component\Serialization\Json instead?

Reply:
Not, if we want to keep JSON_THROW_ON_ERROR